yb-voyager/cmd/root.go

```diff
@@ -379,23 +385,26 @@ func metaDBIsCreated(exportDir string) bool {
 	return utils.FileOrFolderExists(filepath.Join(exportDir, "metainfo", "meta.db"))
 }
 
-func setControlPlane(cpType string) {
+func setControlPlane(cpType string) error {
 	switch cpType {
 	case "":
 		log.Infof("'CONTROL_PLANE_TYPE' environment variable not set. Setting cp to NoopControlPlane.")
 		controlPlane = noopcp.New()
 	case YUGABYTED:
 		ybdConnString := os.Getenv("YUGABYTED_DB_CONN_STRING")
 		if ybdConnString == "" {
-			utils.ErrExit("'YUGABYTED_DB_CONN_STRING' environment variable needs to be set if 'CONTROL_PLANE_TYPE' is 'yugabyted'.")
+			return fmt.Errorf("yugabyted-db-conn-string config param (or YUGABYTED_DB_CONN_STRING environment variable) needs to be set if control plane type is 'yugabyted'.")
 		}
 		controlPlane = yugabyted.New(exportDir)
 		log.Infof("Migration UUID %s", migrationUUID)
 		err := controlPlane.Init()
 		if err != nil {
-			utils.ErrExit("ERROR Failed to initialize the target DB for visualization. %s", err)
+			return fmt.Errorf("initialize the target DB for visualization. %s", err)
 		}
+	default:
+		return fmt.Errorf("invalid value of control-plane-type (or CONTROL_PLANE_TYPE env var): %q. Allowed values: %v", cpType, []string{YUGABYTED})
```

Comment:
Here probably we can take the genralized route? Like just this sentence:
`invalid value of control plane type: %q. Allowed values: %v`
Again I agree that it seems a little weird mentioning it in this way.